.github/workflows/ci.yml

```diff
@@ -37,3 +37,35 @@ jobs:
 
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v1
+
+  release:
+
+    needs: build
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK 15
+      uses: actions/setup-java@v1
+      with:
+        java-version: 15
+
+    - name: Cache Local Maven Repo
+      uses: actions/cache@v2.1.2
+      with:
+        path: ~/.m2/repository
+        key: maven-repo
+
+    - name: Release
+      uses: qcastel/github-actions-maven-release@v1.12.16
```

Comment:
This doesn't change any version number - right? So I just put the right version number in the commit on the main branch und the release goes with that?

Reply:
That's correct. It uses the official maven release functionality. It only provides options for incrementing the major or minor version instead of the patch version for the next development version (2nd automated commit).

Reply:
But we don't use the options to increase the versions - right? I would like to increment the versions by myself.

Reply:
No, we don't. I don't see any benefits compared setting the development version manually with a corresponding description as commit message.